### PR TITLE
Pl fixes 1

### PR DESCRIPTION
**Pull request by pldespaigne**

### CODE REVIEW

File: apps/festival/festival/src/app/marketplace/event/session/session.component.ts [hunk 1]
```diff
@@ -70,11 +72,13 @@ export class SessionComponent implements OnInit, OnDestroy {
       }
 
       if (event.type === 'screening') {
+        this.dynTitle.setPageTitle(event.title, 'Screening');
         if (!!(event.meta as Screening).titleId) {
           const movie = await this.movieService.getValue(event.meta.titleId as string);
           this.screeningFileRef = movie.promotional.videos?.screener?.ref ?? '';
         }
       } else if (event.type === 'meeting') {
+        this.dynTitle.setPageTitle(event.title, 'Meeting');
```
Comment (fritzschoff) [inline apps/festival/festival/src/app/marketplace/event/session/session.component.ts:81]: do it after the if else statement if you are using event.title in both if else statements

Comment (fritzschoff) [reply apps/festival/festival/src/app/marketplace/event/session/session.component.ts:81]: so you need to do it only once

File: libs/ui/src/lib/form/table/form-table.component.scss [hunk 2]
```diff
@@ -35,21 +33,6 @@ mat-paginator {
...
-  height: 10px;
-}
-
-::-webkit-scrollbar-track {
-  height: 10px;
-  background-color: var(--active);
-  opacity: 0.5;
-}
-
-::-webkit-scrollbar-thumb {
```
Comment (fritzschoff) [inline libs/ui/src/lib/form/table/form-table.component.scss:48]: so you don't style it anymore?

**Review by fritzschoff — APPROVED** [review]

File: libs/ui/src/lib/form/table/form-table.component.scss (hunk 2 above)
Comment (pldespaigne) [reply libs/ui/src/lib/form/table/form-table.component.scss:48]: yup

**Pull request merged**